tests/testCameraUtils.py

```diff
@@ -1531,13 +1712,88 @@ def testDistortion(self):
                                             includeDistortion=True)
 
         # just verify that the distorted versus undistorted coordinates vary in the 4th decimal
-        with self.assertRaises(AssertionError) as context:
-            numpy.testing.assert_array_almost_equal(arcsecFromRadians(xu), arcsecFromRadians(xd), 4)
+        self.assertRaises(AssertionError,
+                          np.testing.assert_array_almost_equal,
+                          arcsecFromRadians(xu),
+                          arcsecFromRadians(xd),
+                          4)
 
-        with self.assertRaises(AssertionError) as context:
-            numpy.testing.assert_array_almost_equal(arcsecFromRadians(yu), arcsecFromRadians(yd), 4)
+        self.assertRaises(AssertionError,
+                          np.testing.assert_array_almost_equal,
+                          arcsecFromRadians(yu),
+                          arcsecFromRadians(yd),
+                          4)
 
 
+class CornerTest(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        cameraDir = getPackageDir('sims_coordUtils')
+        cameraDir = os.path.join(cameraDir, 'tests', 'cameraData')
+        cls.camera = ReturnCamera(cameraDir)
+
+    def testCornerPixels(self):
+        """
+        Test the method to get the pixel coordinates of the corner
+        of a detector
+        """
+        det_name = self.camera[0].getName()
+        corners = getCornerPixels(det_name, self.camera)
+        self.assertEqual(corners[0][0], 0)
+        self.assertEqual(corners[0][1], 0)
+        self.assertEqual(corners[1][0], 0)
+        self.assertEqual(corners[1][1], 3999)
+        self.assertEqual(corners[2][0], 3999)
+        self.assertEqual(corners[2][1], 0)
+        self.assertEqual(corners[3][0], 3999)
+        self.assertEqual(corners[3][1], 3999)
+        self.assertEqual(len(corners), 4)
```

Comment:
Might as well make this a test for shape instead of just length and make sure each row is only a list of 2?